cpu/fe310/include/periph_cpu.h

```diff
@@ -67,9 +61,9 @@ typedef uint8_t gpio_t;
  */
 typedef struct {
     uint32_t addr;          /**< UART control register address */
-    gpio_t rx;              /**< RX pin */
-    gpio_t tx;              /**< TX pin */
-    plic_source isr_num;    /**< ISR source number */
+    gpio_t   rx;              /**< RX pin */
+    gpio_t   tx;              /**< TX pin */
+    irqn     isr_num;    /**< ISR source number */
```

Comment:
Please keep the doxygen comment aligned:
```suggestion
    gpio_t   rx;            /**< RX pin */
    gpio_t   tx;            /**< TX pin */
    irqn     isr_num;       /**< ISR source number */
```

Reply:
Aligned them